PxAPI-2.yml

```diff
@@ -310,40 +310,39 @@ components:
             discontinued:
               type: boolean
               nullable: true
-    Paginated:
+
+    PageInfo:
       type: object
       properties:
-        page:
-          description: Pagination properties
-          title: PaginationProperties
-          type: object
-          properties:
-             pageNumber:
-              type: integer
-              format: int32
-             pageSize:
-              type: integer
-              format: int32
-              example: 33
-             totalElements:
-              type: integer
-              format: int32
-             totalPages:
-              type: integer
-              format: int32
-        links:
-          type: array
-          items:
-             $ref: '#/components/schemas/Link'
+        pageNumber:
+          type: integer
+          format: int32
+          minimum: 1
+        pageSize:
+          type: integer
+          format: int32
+          example: 20
+          minimum: 5
+          maximum: 500
+        totalElements:
+          type: integer
+          format: int32
+          minimum: 0
+        totalPages:
+          type: integer
+          format: int32
+          minimum: 0
+

```

Comment:
No links to next , prev page?

Reply:
Links were was removed by mistake, should be back now.